provisioning_templates/provision/preseed_default.erb

```diff
@@ -8,9 +8,16 @@ oses:
 %>
 <%
   proxy_string = host_param('http-proxy') ? " http://#{host_param('http-proxy')}:#{host_param('http-proxy-port')}" : ''
+  ansible_enabled = @host.respond_to?(:all_ansible_roles)
```

Comment:
should this also check whether some roles are assigned? I think this returns true for each host, after ansible plugin is installed

Reply:
That helps, if you decide to add roles later.

Reply:
... or if you want to use ansible for remote-execution tasks.

Reply:
Oh, that's a good point. So, how about we add `plugin_present?` macro in core? That's an easy addition and much cleaner approach :-) I can help with that if the need be.

Reply:
I like the idea of `plugin_present?`and we should definitely do that, but i am not sure we should wait here, as this one is working with today's foreman.
(I begun to dislike interdependent PR's a bit...)

Reply:
https://github.com/theforeman/foreman/pull/6543

Reply:
I promise I'll help to get this PR in. Once we have something in, we never refactor...

Reply:
@ares Thanks for keeping that promise.
Ready for rereview.